ui/decredmaterial/switch.go

```diff
@@ -28,8 +38,12 @@ type SwitchButtonText struct {
 	selected                           int
 }
 
-func (t *Theme) Switch(swtch *widget.Bool) Switch {
-	return Switch{material.Switch(t.Base, swtch)}
+func (t *Theme) Switch(swtch *widget.Bool) *Switch {
+	sw := &Switch{
+		Switch: swtch,
+	}
+	sw.Enabled, sw.Disabled, sw.thumbColor = t.Color.Primary, t.Color.InactiveGray, t.Color.Surface
```

Comment:
Disabled and inactive states are mixed up here. Add an inactive state for when the switch is inactive and a disabled state for when user interaction is disabled on the switch.